### PR TITLE
Close: #435 - Change Operation Code difficulty level to 'medium'.

### PR DESCRIPTION
**Pull request by wuworkshop**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

This resolves issue #435 - changing Operation Code's difficulty level from 'easy' to 'medium'.

### CODE REVIEW

**Comment by swapagarwal** [comment]
Thanks @wuworkshop! 👍

File: data.json [hunk 1]
```diff
@@ -316,10 +316,10 @@
   },
   {
     "name": "Operation Code",
-    "difficulty": "easy",
-    "description": "Close 3 issues, get a t-shirt. Make 2 PRs and get stickers.",
+    "difficulty": "medium",
+    "description": "Resolve 3 issues, get a t-shirt. Merge 2 PRs and get stickers.",
     "reference": "https://github.com/OperationCode/START_HERE",
```
Comment (aslafy-z) [inline data.json]: I guess we can change that image but we would need to rehost it on imgur for example.

File: data.json [hunk 2]
```diff
@@ -75,7 +75,7 @@
     "name": "CircleCI",
     "difficulty": "medium",
     "description": "Participate in #Orbtoberfest with CircleCI and claim limited edition swag! 1 - 3 pull requests: Limited-Edition Sticker, 4+ pull requests: Limited-Edition T-shirt",
-    "reference": "https://hacktoberfest.circleci.com/#/" ,
+    "reference": "https://hacktoberfest.circleci.com/#/",
```
Comment (aslafy-z) [inline data.json]: Let's correct this url at the same time. Please drop the trailing #/

File: data.json (hunk 1 above)
Comment (swapagarwal) [reply data.json]: Sorry I merged early. 🙈

**Pull request merged**